tests/test_mongodb_connection.py

```diff
@@ -0,0 +1,36 @@
+import unittest
+import hydra
+import mongomock
+
+from core_lib.data_transform.result_to_dict import result_to_dict
+from core_lib.helpers.config_instances import instantiate_config
+from core_lib.connection.mongodb_connection_registry import MongoDBConnectionRegistry
+hydra.core.global_hydra.GlobalHydra.instance().clear()
+hydra.initialize(config_path='./test_data/test_config')
+configfile = "test_mongodb.yaml"
+config = hydra.compose(configfile)
+
+
+class TestMongoDBConnection(unittest.TestCase):
+
+    @mongomock.patch(servers=((config.core_lib.mongodb.config.url.host, config.core_lib.mongodb.config.url.port), ))
+    def test_instantiate(self):
+        mongodb = instantiate_config(config.core_lib.mongodb)
+        self.assertIsInstance(mongodb, MongoDBConnectionRegistry)
+        with mongodb.get() as client:
+            collection = client.testing_collection.example
+            data = result_to_dict(collection.find())
+            self.assertEqual(type(data), type([1, 2, 3]))
+
+            user_id = collection.insert_one({'name': 'ansh', 'age': 18}).inserted_id

```

Comment:
you can create a variable for this object foe example

```python
test_insert_data = {'name': 'ansh', 'age': 18}

# and in the asserts below you can make use of the same
self.assertEqual(entry['name'], test_insert_data['name'])
self.assertEqual(entry['age'], , test_insert_data['age'])
```

this will make sure you are testing the same values that you are inserting, ad will eliminate the chance making typo mistakes in strings